### ISSUE TITLE
Pre-built binaries for Electron 3

### ISSUE THREAD

**Issue by rbeeger** [galkahana/HummusJS#344]
Please add pre-built binaries for Electron 3.

**Comment by galkahana** [comment]
cheers.

**Comment by hhaidar** [comment]
I've opened a PR for this.

@galkahana electron v4 is coming out soon too, it might make sense to drop support for v1 at this point.

**Comment by galkahana** [comment]
@hhaidar let's drop 1 once 4 gets out. makes sense?

**Comment by galkahana** [comment]
yeah. this is the error i saw on linux/mac. see here - https://travis-ci.org/galkahana/HummusJS/builds/456039328 - in allowed failures.

the windows one goes ok - https://ci.appveyor.com/project/galkahana/hummusjs/builds/20358304

**Comment by julianhille** [comment]
#410 might solve these
